packages/opentelemetry-instrumentation-langchain/opentelemetry/instrumentation/langchain/callback_handler.py

```diff
@@ -384,7 +454,7 @@ def on_chat_model_start(
 
         name = self._get_name_from_callback(serialized, kwargs=kwargs)
         span = self._create_llm_span(
-            run_id, parent_run_id, name, LLMRequestTypeValues.CHAT, metadata=metadata
+            run_id, parent_run_id, name, LLMRequestTypeValues.CHAT, metadata=metadata, serialized=serialized
```

Comment:
instead of propagating serialized just to get the vendor inside make your detect_vendor get serialized as an input
```suggestion
vendor = self._detect_vendor_name(serialized)
span = self._create_llm_span(
            run_id, parent_run_id, name,vendor, LLMRequestTypeValues.CHAT, metadata=metadata)
```